cgi-bin/DW/Media.pm

```diff
@@ -110,9 +110,15 @@ sub upload_media {
 
     # set the security
     my $sec = $opts{security} || 'public';
+    $sec = 'usemask' if $sec =~ /^(?:friends|access)$/;
```

Comment:
The standard way we handle this access is,

```perl
if ( $sec =~ /^(?:friends|access)$/ ) {
    $sec = 'usemask';
    $allowmask = 1;
}
```

I know that your code _should_ be safe because it uses defined-or, but I'd much rather have the code be extremely explicit about the access being set and not touch it if it's already `usemask`.